outlines/deployment.md

```diff
@@ -0,0 +1,69 @@
+# Deployment, monitoring and analytics
+
+1. Deploying a web application
+  1. A web application is not necessarily like any other piece of released software
+     1. Can be updated frequently
+     2. Still QA is important!
+  2. Deployment environments: production vs staging vs development
+    1. Staging should be as close as possible to production
+  3. Meteor ENV + settings
```

Comment:
This overlaps with the security chapter - I think the bulk of settings belongs here, so we should slim down that part